modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaMicronautAbstractCodegen.java

```diff
@@ -138,6 +141,7 @@ public JavaMicronautAbstractCodegen() {
         cliOptions.add(new CliOption(OPT_APPLICATION_NAME, "Micronaut application name (Defaults to the " + CodegenConstants.ARTIFACT_ID + " value)").defaultValue(appName));
         cliOptions.add(CliOption.newBoolean(USE_BEANVALIDATION, "Use BeanValidation API annotations", useBeanValidation));
         cliOptions.add(CliOption.newBoolean(USE_OPTIONAL, "Use Optional container for optional parameters", useOptional));
+        cliOptions.add(CliOption.newBoolean(OPT_USE_SEALED, "Use abstract sealed classes for subtypes with a discriminator", useSealed));
```

Comment:
Can you add `"(Experimental) "` to the beginning of the description, as the way it works may change.